src/i18n/locales/et.json

```diff
@@ -188,7 +188,10 @@
     "institution_settings": "Asutuse sätted",
     "technical_settings": "Tehnilised sätted",
     "collapse_menu": "Kitsendan menüü",
-    "manual": "Kasutusjuhend"
+    "manual": "Tõlkevärava kasutusjuhend",
```

Comment:
**[blocker]** 

We should add a new string for this, or a new one for the sidebar. In sidebar the item should be called "Kasutusjuhend"